src/input_sudoku.c

```diff
@@ -0,0 +1,57 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+void input_sudoku(int arr[2]);
+
+
+/*
+* File name: input_sudoku.c
+*
+* Brief: This function takes input from the user.
+*
+* Description: The first input asks for the difficuly level.The second input asks for the number of sudokus that user wants to generate.
+*
+* Warning: Do not enter any charaters or symbols because the program will crash. This bug will be solved before release 2.
+*
+* Author: Rishith Rao - rishithrao@cmail.carleton.ca
+*
+**/
+void input_sudoku(int arr[2]) { 
+{
+  int diff_level,no_sudoku;
+ /*This input askes for the difficulty level*/
+  while(1){
+    printf("Enter difficulty level as(0,1,2 or 3):\nEasy:      0\nMedium:    1\nHard:      2\nVery Hard: 3\n");
+    scanf("%d", &diff_level);
+    if (diff_level != 0 && diff_level !=1 && diff_level !=2 && diff_level !=3) 
```

Comment:
Stay consistent with the bracket position: it should look like this:
if{
...
}